src/config.c

```diff
@@ -1079,7 +1079,8 @@ dictType optionToLineDictType = {
     NULL,                       /* val dup */
     dictSdsKeyCaseCompare,      /* key compare */
     dictSdsDestructor,          /* key destructor */
-    dictListDestructor          /* val destructor */
+    dictListDestructor,         /* val destructor */
+    NULL                        /* allow to expand */
```

Comment:
Note - while arguably good practice, this is not technically needed.  C is defined to fill the remainder of a struct with NULL when a static initializer like this is used.  If you're worried about touching too much code, this could be left out.

Reply:
yes. but some compilers emit warnings for that, since it's just some 20 lines, we can swallow that frog.